client/src/pages/Train.jsx

```diff
@@ -222,21 +276,35 @@ class Train extends Component {
       result: { good: [], bad: [] },
       error: null,
       questionNum: 0,
+      system: null,
+      difficulty: null,
```

Comment:
Comme le composant est assez complexe, on peut préciser que ce sont des filtres pour une question de lisibilité
```suggestion
      systemFilter: null,
      difficultyFilter: null,
```